faithcomesbyhearing/dbp
# FCBHDBP-525 DBP: validate request parameter "sort_by" before applying it to the DB sort

# Description
Add new logic to validate that the value of the query parameter 'sort_by' for the endpoints Notes, Bookmarks, and Highlights must correspond to a column of the respective entity according to the endpoint.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-525

## How Do I QA This
We should run all postman test of the following folder and they have to pass. Also, we must be logged in before.
- https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/folder/12519377-20fd4623-f8a3-4618-95d8-33833052285d
